sonic_platform_base/sonic_xcvr/api/public/sff8636.py

```diff
@@ -360,3 +360,35 @@ def get_lpmode_support(self):
 
     def get_power_override_support(self):
         return not self.is_copper()
+
+    def set_lpmode(self, lpmode):
+        '''
+        This function sets LPMode for the module.
+
+        Args:
+            lpmode (bool): False means "set to high power", True means "set to low power"
```

Comment:
I want to suggest  using "False means LPMode Off, True means LPMode On", the reason is like below:
if we check the output of  CLI "sfputil show lpmode", we can see it is using 'on/off' to indicate the status, it will make it consistent across layers.

> ```
> admin@sonic:# sfputil show lpmode 
> Port         Low-power Mode
> -----------  ----------------
> Ethernet180  Off
> Ethernet216  On
> ```

Reply:
good suggestion. update it.